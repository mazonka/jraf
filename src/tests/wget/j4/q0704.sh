#!/bin/sh

cwd=`sh execdir.sh`
wr=$cwd/wroot
echo ""
echo "=="
#ls -1 $wr/home
rm -rf $wr



