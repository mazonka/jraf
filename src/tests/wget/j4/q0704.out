OK /.jraf.sys/users : JRAF_ERR bad email
==
